install_builder/deadline-cloud-for-unreal-engine.xml

```diff
@@ -41,6 +41,34 @@
         <actionList>
             <setInstallerVariable name="all_components" value="${all_components} deadline_cloud_for_unreal_engine"/>
             <setInstallerVariable name="unreal_deps_platform" value="windows"/>
+            <setInstallerVariable name="unreal_install_path" value="C:\Program Files\Epic Games\UE_5.4"/>
+            <registryFind>
+                <keyPattern>*5.*</keyPattern>
+                <namePattern>InstalledDirectory</namePattern>
+                <findAll>1</findAll>
+                <rootKey>HKEY_LOCAL_MACHINE\SOFTWARE\EpicGames\Unreal Engine</rootKey>
+                <searchDepth>1</searchDepth>
+                <variable>unreal_registry_list</variable>
+            </registryFind>
+            <foreach>
+                <variables>key name value</variables>
+                <values>${unreal_registry_list}</values>
```

Comment:
So to clarify, each unreal installation (major.minor?) will list its installation directory here, and we're just presenting the last one that we find to the user? They can still override and choose a different one?

Additionally, how are those registry values sorted? Latest will always be last?

Reply:
Yep!  Each version goes into the registry here, most recently installed version last, and what we're gathering here is just the suggested default which we still present them with the option to override with some other folder, as long as they point to a plugins folder which exists.

Reply:
perfect, thanks!